### PR TITLE
Allcoredocs

### PR DESCRIPTION
**Pull request by rowanmiller**
Looking at Query/* docs is lower priority as @anpete wrote them and I reviewed them.

I think the ones that need review the most are 
* DbContextOptionsBuilder
* WarningBehavior
* WarningsConfigurationBuilder

### CODE REVIEW

File: src/Microsoft.EntityFrameworkCore/DbContextOptionsBuilder.cs [hunk 1]
```diff
@@ -129,6 +129,22 @@ public virtual DbContextOptionsBuilder UseInternalServiceProvider([CanBeNull] IS
         public virtual DbContextOptionsBuilder EnableSensitiveDataLogging()
             => SetOption(e => e.IsSensitiveDataLoggingEnabled = true);
 
+        /// <summary>
+        ///     Configures the behavior when a warning is raised by Entity Framework. You can set a default behavior and behaviors for
```
Comment (anpete) [inline src/Microsoft.EntityFrameworkCore/DbContextOptionsBuilder.cs]: "warning is generated"?

Comment (anpete) [inline src/Microsoft.EntityFrameworkCore/DbContextOptionsBuilder.cs]: Maybe "Configures the runtime behavior of warnings generated by Entity Framework"?

Comment (rowanmiller) [inline src/Microsoft.EntityFrameworkCore/DbContextOptionsBuilder.cs]: Yep I like that, updated to this language for all the warning related APIs

**Comment by divega** [comment]
:shipit:

**Pull request merged**